build.sbt

```diff
@@ -134,6 +134,8 @@ lazy val docs = (project in file("docs"))
       "empty" -> "",
       "version" -> version.value
     ),
+    paradoxNavigationDepth := 1,
```

Comment:
would you consider 2 for quicker navigation?

Reply:
I guess it could make sense:


![toc-depth2](https://user-images.githubusercontent.com/9868/43765969-95b1aa8a-9a31-11e8-91b6-a1460c169c1c.png)

Reply:
Yes plz

Reply:
Done